services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -1214,8 +1218,8 @@ class HakemusServiceITest(
             inner class AccompanyingJohtoselvitys {
 
                 private var kaivuilmoitusHakemusId: Long = 0
-                private val cableReportAlluId: Int = alluId
-                private val excavationNotificationAlluId: Int = alluId + 1
+                private val cableReportId: Int = alluId
```

Comment:
Would `johtoselvitysId` be more suitable as there is `kaivuilmoitusId` now as well in Finnish?